Tests/UnitTests/Application/MainCoordinatorTests.swift

```diff
@@ -17,6 +18,7 @@ final class MainCoordinatorTests: XCTestCase {
     var mockUserStore: UserStorage!
     var mockTokenVerifier: MockTokenVerifier!
     var mockUpdateService: MockAppInformationService!
+    var walletAvailabilityService = WalletAvailabilityService()
```

Comment:
Should a mock for WalletAvailabilityService be used in MainCoordinator? Or is that not worth setting up do we think?

Reply:
I don't even think I ended up using this in the end - might just need to remove it altogether 😅 Thanks!